systemtest/produce_consume_basic_test.go

```diff
@@ -169,17 +170,18 @@ func TestProduceConsumeMultipleTopics(t *testing.T) {
 				events:  events,
 				records: &records,
 				timeout: timeout,
-				producer: newPubSubLiteProducer(ctx, t, pubsublite.ProducerConfig{
-					Topics:      topics,
+				producer: newPubSubLiteProducer(t, pubsublite.ProducerConfig{
 					Logger:      logger,
 					Encoder:     json.JSON{},
 					TopicRouter: topicRouter,
 					Sync:        sync,
 				}),
 				consumer: newPubSubLiteConsumer(ctx, t, pubsublite.ConsumerConfig{
-					Logger:        logger,
-					Decoder:       json.JSON{},
-					Subscriptions: pubSubLiteSubscriptions(topics...),
+					Logger:  logger,
+					Decoder: json.JSON{},
+					Topics:  topics,
+					Project: googleProject,
+					Region:  googleRegion,
```

Comment:
These two will always need to be set to `googleProject` and `googleRegion`. Should we do that in `newPubSubLiteConsumer`?

Reply:
SGTM, will update